### ISSUE TITLE
Update survey to include question asking which showcase the participant went through

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/BC-Wallet-Demo#106]
The current survey doesn't ask which showcase people just completed. Feedback will be hard to parse without that knowledge. 
Please add a question that asks which showcase they just completed.

**Update**
Question: 
Who did you follow in the showcase?
Multiple choice answers:
- Joyce the lawyer
- Alice the student

**Comment by jeznorth** [comment]
@hannahgov can you confirm if this is still relevant? If not, we can close.

**Comment by hannahgov** [comment]
> [@hannahgov](https://github.com/hannahgov) can you confirm if this is still relevant? If not, we can close.

Still not in the current survey.
